pkg/controller/bundlec/bundle_sync_task.go

```diff
@@ -373,9 +401,27 @@ func (st *bundleSyncTask) handleProcessResult(retriable bool, processErr error)
 			}
 		}
 
-		bundleUpdated = updateBundleCondition(st.bundle, &inProgressCond) || bundleUpdated
-		bundleUpdated = updateBundleCondition(st.bundle, &readyCond) || bundleUpdated
-		bundleUpdated = updateBundleCondition(st.bundle, &errorCond) || bundleUpdated
+		inProgressUpdated := updateBundleCondition(st.bundle, &inProgressCond) || bundleUpdated
+		readyUpdated := updateBundleCondition(st.bundle, &readyCond) || bundleUpdated
+		errorUpdated := updateBundleCondition(st.bundle, &errorCond) || bundleUpdated
```

Comment:
Remove `|| bundleUpdated` here and above.

Reply:
Oops - vim replacement errors